code/datums/components/machine_power_modifier.dm

```diff
@@ -0,0 +1,30 @@
+/datum/component/machine_power_modifier
+	dupe_mode = COMPONENT_DUPE_UNIQUE
+	var/idle_power_multiplier = 1
+	var/active_power_multiplier = 1
+
+/datum/component/machine_power_modifier/Initialize(idle_power_multiplier, active_power_multiplier)
```

Comment:
```suggestion
/datum/component/machine_power_modifier/Initialize(idle_power_multiplier = 1, active_power_multiplier= 1)
```
Good to include defaults in the parameters, otherwise you're setting them to `null` if someone doesn't include them